### ISSUE TITLE
decoupling nucleus in MTBLS1 Example 5 (Proton NMR) set to true ? Should be false ?

### ISSUE THREAD

**Issue by DSchober** [nmrML/nmrML#183]
I was made aware by our NMR expert (A. Porzel) that in MTBLS1 (http://nmrml.org/examples/5/) which is  Proton NMR, the **decoupling nucleus should be set 'false'**.
Does anyone know if this hints to some systematic error in the vendor to nmrML converter ? If so, it should probably be corrected.
Best,
  Daniel.

**Comment by rsalek** [comment]
Where is this? On proton channel or X

**Comment by DSchober** [comment]
Dear all. 
According to Andrea Porzel, the following is the problem:
the pulse sequence used in "Example 5: Human urine sample spectra from the MTBLS1 study" is noesypr1d:
 <userParam name="Pulse Program" value="noesypr1d"/>
The transmitter nulcleus is 1H:
<acquisitionNucleus cvRef="CHEBI" accession="CHEBI_49637" name="hydrogen atom"/>
[Andrea Porzel would prefer "1H" instead of "hydrogen atom", but since we use chebi I guess for now we have to live with this unintuitive wording]

The decoupler nuleus is also 1H:
 <decouplingNucleus cvRef="CHEBI" accession="CHEBI_49637" name="hydrogen atom"/>
Decoupling is showed as on:
<DirectDimensionParameterSet decoupled="true" numberOfDataPoints="65536">
which makes no sense, as only one channel should have been used (as far as I know the Bruker software - ask a Bruker user, please, for verification) and an 1H spectrum - with exception of selective decoupling - is never 1H (broad band) decoupled.

In example 2 (http://nmrml.org/examples/2/), the sequence is "zg" (simple 1D measurement),  decoupled is correctly = false, but here
 <decouplingNucleus cvRef="NMRCV" accession="NMR:1000055" name="off resonance decoupling"/>
does not really make sense.

In example 1 (http://nmrml.org/examples/1/), a Varian expample, it's ok:
 <DirectDimensionParameterSet numberOfDataPoints="65536" decoupled="false">
 <acquisitionNucleus name="1H" cvRef="ChEBI" accession="CHEBI_29236"/>
 <decouplingNucleus name="1H" cvRef="ChEBI" accession="CHEBI_29236"/>,
but I'm missing the name of the pulse sequence (seqfil = "s2pul").

**Comment by proccaserra** [comment]
@DSchober: CHEBI (like many ontologies as you may know) allows for synonyms. Please have a look at CHEBI  for 'hydrogen atom' (i will give you some help here: http://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI:49637).
you will see that 1H is one of the vetted alternative labels.

I pointed this too you already on repeated occasions.

So reusing classes from CHEBI is good practice (instead of minting a new class for the same thing in a different namespace) and you can still have your preferred label.
In other words, you have your cake and you can eat it !

**Comment by DSchober** [comment]
@proccaserra: Chebis notion of 'synonym' is underspecifies as the allow for colloquial synonyms like "Wasserstoff" which is NOT an exact synonym for hydrogen atom, as it usually refers to a H2 molecule. 1H synonym would be correct, but in general alowing this will cause semantic blur.
But lets stick to the more important issue of a potential systematic error in converting Bruker decoupling specification. It seems to work fine with Varian data. @rsalek , could you look into this ?
